crates/router/src/connector/utils.rs

```diff
@@ -74,6 +74,10 @@ pub trait RouterData {
     fn get_quote_id(&self) -> Result<String, Error>;
 }
 
+pub fn payment_method_error_message(connector: &str) -> String {
```

Comment:
rename this method to `get_unimplemented_payment_method_error_message()` you can take this in next PR